GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -231,17 +247,38 @@ private bool TryRunUpgrade(out Version newVersion, out string consoleError)
             {
                 this.mount = false;
 
-                consoleError = errorMessage;
+                consoleError = error;
                 return false;
             }
 
+            string postInstaller;

```

Comment:
Same here - I would hold off on adding the new pre / post install logic in this PR.